jdaviz/configs/cubeviz/helper.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, *args, **kwargs):
                                handler=self._set_spectrum_x_axis)
 
     def _set_spectrum_x_axis(self, msg):
-        if msg.viewer_id != "cubeviz-3":
+        if msg.viewer_id != "cubeviz-2":
             return
         viewer = self.app.get_viewer("spectrum-viewer")
```

Comment:
```suggestion
        viewer = self.app.get_viewer("spectrum-viewer")
        if msg.viewer_id != viewer.reference_id:
            return
```

this might be slightly more expensive, but would remove one place that the id is hardcoded 🤷

Reply:
I don't mind a tiny performance hit to make this robust against changing the reference ID.